### DISCUSSION TITLE
How to use ffmpeg x265 command line encode?

### DISCUSSION THREAD

**Discussion by voska7** [staxrip/staxrip#869] [Q&A]
here is settings i tried, but i getting error. anyone can please tell me what will be the proper way to add it? and how to change -b:v 5000k bitrate value? i could not find any option to edit it. it came with default

https://i.imgur.com/1uZ6VF5.png

**Comment by jessielw** [comment]
The entire command should be wrapped in quotes
`-c:v libx265 -x265-params "all_x265_params_in_these_quotes"`

Why you'd use the ffmpeg command line to encode when the program supports the official x265 tool for encoding is beyond me though. Instead of using those parameters you could literately just use the x265 encoder tool, officially...

**Comment by voska7** [comment]
thanks for the info. i need to add these extra parameters
https://pastebin.com/HNKhKLuP

can we use it in official x265 custom field? if so how can we add?
https://i.postimg.cc/52yncMQC/image.png

also can we use in official x265 custom field?

https://i.imgur.com/N51uPQi.png

**Reply by jessielw** [reply]
X265 params is the command for FFMPEG. To allow ffmpeg to use x265 commands. 

You need to just use the actual commands in the gui, as they are ALL there.
